Tools/AP_Bootloader/bl_protocol.cpp

```diff
@@ -79,7 +81,7 @@
 // RESET		finalise flash programming, reset chip and starts application
 //
 
-#define BL_PROTOCOL_VERSION 		5		// The revision of the bootloader protocol
+#define BL_PROTOCOL_VERSION 		6		// The revision of the bootloader protocol
```

Comment:
do we need to change the version? I'm concerned about older bl clients (mission planner, qgc etc)